### PR TITLE
Add functionality for processing transactions - Closes #881

### LINKED ISSUE #881: Add functionality to process transactions in transaction pool
### Description
Transaction pool needs a function to move verified queue to the pending queue

### PR DESCRIPTION
**Pull request by SargeKhan**
### What was the problem?
The functionality for processing verified transactions was missing.
### How did I fix it?
Added functionality for processing transactions.
### How to test it?
Run tests
### Review checklist

* The PR resolves #881
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: packages/lisk-transaction-pool/src/check_transactions.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export interface TransactionResponse {
+	readonly errors: ReadonlyArray<Error>;
+	readonly id: string;
+	readonly status: Status;
+}
+
+export enum Status {
+	OK = 1,
+	FAIL,
```
Comment (shuse2) [inline packages/lisk-transaction-pool/src/check_transactions.ts:20]: FAIL should be 0?

File: packages/lisk-transaction-pool/src/transaction_pool.ts [hunk 2]
```diff
@@ -325,6 +311,63 @@ export class TransactionPool {
 		);
 	}
 
+	private async processVerifiedTransactions(): Promise<CheckTransactionsResponse | void> {
```
Comment (shuse2) [inline packages/lisk-transaction-pool/src/transaction_pool.ts]: would it be possible to return the same type? maybe return something like `{ failed: [], passed: [] }`?

File: packages/lisk-transaction-pool/src/transaction_pool.ts [hunk 3]
```diff
@@ -325,6 +311,63 @@ export class TransactionPool {
...
+		const transactionsInReadyQueue = this._queues.ready.size();
+		const transactionsInVerifiedQueue = this._queues.verified.size();
+		if (
+			this._verifiedTransactionsProcessingLimitPerInterval >
+				transactionsInReadyQueue &&
+			transactionsInVerifiedQueue > 0
+		) {
```
Comment (shuse2) [inline packages/lisk-transaction-pool/src/transaction_pool.ts:321]: prefer early return

**Review by shuse2 — DISMISSED** [review]

File: packages/lisk-transaction-pool/src/check_transactions.ts (hunk 1 above)
Comment (SargeKhan) [reply packages/lisk-transaction-pool/src/check_transactions.ts:20]: I've duplicated the enum implemented in lisk-transactions module. With this implementation, the value of `FAIL` will be 2. Maybe @mitsujutsu can shed some light here.

Comment (shuse2) [reply packages/lisk-transaction-pool/src/check_transactions.ts:20]: I will create another issue to solve both of them at the same time =)

**Review by shuse2 — APPROVED** [review]

**Review by mitsuaki-u — APPROVED** [review]

**Pull request merged**